cromshell

```diff
@@ -277,35 +298,67 @@ function execution-status-count() {
 # Bring up a browser window to view timing information on the job.
 function timing() {
   echo "Opeining timing information in a web browser for job ID: ${1}"
-  open ${CROMWELL_URL}/api/workflows/v1/${1}/timing
+  open ${2}/api/workflows/v1/${1}/timing
   return $?
 }
 
 # Cancel a running job.
 function abort() { 
-  response=$(curl --connect-timeout $CURL_CONNECT_TIMEOUT -X POST --header "Content-Type: application/json" --header "Accept: application/json" "${CROMWELL_URL}/api/workflows/v1/${1}/abort")
+  response=$(curl --connect-timeout $CURL_CONNECT_TIMEOUT -X POST --header "Content-Type: application/json" --header "Accept: application/json" "${2}/api/workflows/v1/${1}/abort")
   local r=$?
   echo $response  
   return $r
 }
 
+# List all jobs submitted in ${CROMWELL_SUBMISSIONS_FILE}
+function list() {
+	cat ${CROMWELL_SUBMISSIONS_FILE}
+	return $?
+}
+
+# Get the root log folder from the cloud and put it in the metadata folder for this run
+function fetch() {
```

Comment:
should we rename this fetch-logs?  It's not clear what it's going to fetch

Reply:
I was wondering about this.  Sure.